### ISSUE TITLE
[FEAT] Add releases to F-Droid

### ISSUE THREAD

**Issue by jeffmcneill** [PhilippC/keepass2android#2650]
For greater discover, please add releases to F-Droid. Stay awesome!

**Comment by SotchNam** [comment]
Wanted to install app, first instinct was to use F-Droid, really hope it well be there soon.

**Comment by PhilippC** [comment]
duplicate of #38

**Comment by jeffmcneill** [comment]
Yeah, and still blocked on fdroid's build platform, sadly https://gitlab.com/fdroid/fdroiddata/-/issues/1529

**Comment by PhilippC** [comment]
I definitely would be happy if they would support it, too.
